prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -700,6 +703,9 @@ def build_transmission(method="line2sub", **kwargs):
     # Add transformers, and calculate rating and impedance for all branches
     transformers = create_transformers(bus)
     transformers["type"] = "Transformer"
+    transformers.index = pd.RangeIndex(
+        ac_lines.index.max() + 1, ac_lines.index.max() + len(transformers) + 1
```

Comment:
We can factor out `ac_lines.index.max()`

Reply:
Done.